README.md

```diff
@@ -66,4 +66,5 @@ Each requirement is configured with the options provided by that Chart. Please c
 | `pgadmin.enabled`                                                           | Enable PgAdmin                                                                                                     | `true`                          |
 | `openldap.enabled`                                                          | Enable openldap                                                                                                    | `true`                          |
 | `phpldapadmin.enabled`                                                      | Enable phpLDAPadmin                                                                                                | `true`                          |
+| `ElasticStack.enabled`                                                      | Enable Elastic-stack                                                                                                | `true`                          |
```

Comment:
@AyadiAmen Could you change `ElasticStack` to `elasticstack` ? I think it is better to use lowercases.